applications/CompressiblePotentialFlowApplication/custom_elements/adjoint_analytical_incompressible_potential_flow_element.cpp

```diff
@@ -0,0 +1,191 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//
+//  Main authors:    Iñigo Lopez based on M. Nuñez, A. Geiser, M. Fusseder and R. Rossi work
```

Comment:
shall we remove the "ñ"'s?

Reply:
sure! I will remove it in all the headers of the files changed in this PR. We can do the rest in another PR